### PR TITLE
Enhancement: Add more descriptions

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [ ] adds more descriptions

Follows #2366.
Somewhat related to https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2382#discussion_r92943370.

### CODE REVIEW

File: src/Fixer/Operator/NotOperatorWithSuccessorSpaceFixer.php [hunk 1]
```diff
@@ -50,6 +44,26 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Logical NOT operators (!) should have one trailing whitespace.',
```
Comment (localheinz) [inline src/Fixer/Operator/NotOperatorWithSuccessorSpaceFixer.php]: Seems like a merging `NotOperatorWithSuccessorSpaceFixer` and `NotOperatorWithSpaceFixer` into a single configurable fixer would make sense.

Comment (SpacePossum) [inline src/Fixer/Operator/NotOperatorWithSuccessorSpaceFixer.php]: these are on the list to be merged into `binary_operator_spaces` with config, resolving even more conflicts

File: src/Fixer/Operator/BinaryOperatorSpacesFixer.php [hunk 2]
```diff
@@ -98,17 +100,72 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
...
+$foo = array(
+    "a" => 9000,
+    "abc" => 9001,
+);
+',
+                    array('align_double_arrow' => true)
+                ),
+            ),
+            null,
+            'Aligns or unaligns "=" in consecutive assignments, or "=>" in array initializations',
```
Comment (SpacePossum) [inline src/Fixer/Operator/BinaryOperatorSpacesFixer.php]: not sure if we are going for back ticks or not in cases like these

**Comment by SpacePossum** [comment]
lots of awesome stuff so far, thanks :)

File: src/Fixer/Operator/BinaryOperatorSpacesFixer.php (hunk 2 above)
Comment (localheinz) [inline src/Fixer/Operator/BinaryOperatorSpacesFixer.php]: Adjusted!

**Comment by keradus** [comment]
Thank you @localheinz.

**Comment by SpacePossum** [comment]
thanks @localheinz, having all features of the tool described is even more close now :D

**Comment by localheinz** [comment]
Thank you, @keradus and @SpacePossum!

**Pull request merged**